source/command-logging-and-monitoring/tests/monitoring/unacknowledged-client-bulkWrite.yml

```diff
@@ -52,22 +52,16 @@ tests:
                 update: { $set: { x: 333 } }
           ordered: false
         expectResult:
-          insertedCount:
-            $$unsetOrMatches: 0
-          upsertedCount:
-            $$unsetOrMatches: 0
-          matchedCount:
-            $$unsetOrMatches: 0
-          modifiedCount:
-            $$unsetOrMatches: 0
-          deletedCount:
-            $$unsetOrMatches: 0
-          insertResults:
-            $$unsetOrMatches: {}
-          updateResults:
-            $$unsetOrMatches: {}
-          deleteResults:
-            $$unsetOrMatches: {}
+          $$unsetOrMatches:
+            acknowledged: { $$unsetOrMatches: false }
+            insertedCount: { $$unsetOrMatches: 0 }
+            upsertedCount: { $$unsetOrMatches: 0 }
+            matchedCount: { $$unsetOrMatches: 0 }
+            modifiedCount: { $$unsetOrMatches: 0 }
+            deletedCount: { $$unsetOrMatches: 0 }
+            insertResults: { $$unsetOrMatches: {} }
+            updateResults: { $$unsetOrMatches: {} }
+            deleteResults: { $$unsetOrMatches: {} }
```

Comment:
I decided to leave these additional assertions in place, but I'll note they were never included in the legacy bulkWrite tests ([example](https://github.com/mongodb/specifications/blob/master/source/crud/tests/unified/bulkWrite-deleteMany-hint-unacknowledged.yml#L73C45-L73C112)).

It seemed more important to just allow for an entirely optional result and conditionally check `acknowledged`.